packages/website/src/containers/content/pages/OfficerOverviewPage.tsx

```diff
@@ -98,52 +98,54 @@ export const OfficerOverviewPage: React.FC<OfficerOverviewPageProps> = ({
               {convert(page.roleDescription)}
             </div>
             {/* SOCIAL MEDIA SECTION */}
-            <div css={socialStyles}>
-              <h4>{'Follow ' + page.firstName}</h4>
-              <ul className="Social" css={socialItemsStyles}>
-                {page.facebookUrl ? (
-                  <li>
-                    <a className="Social__link" href={page.facebookUrl}>
-                      <span className="Social__icon Social__icon--facebook">
-                        <span className="u-h">Facebook</span>
-                      </span>
-                      <span className="Social__handle">
-                        {page.facebookUrl || 'Facebook'}
-                      </span>
-                    </a>
-                  </li>
-                ) : null}
+            {page.facebookUrl || page.instagramUrl || page.twitterUsername ? (
+              <div css={socialStyles}>
+                <h4>{'Follow ' + page.firstName}</h4>
```

Comment:
We'll let React handle the concatenation here. Like `<h4>Follow {page.firstName}</h4>`